src/main/java/com/google/codeu/data/Book.java

```diff
@@ -0,0 +1,84 @@
+package com.google.codeu.data;
+
+import java.util.List;
+import java.util.UUID;
+
+public class Book {
+  private final UUID id;
+  private String title;
+  private List<String> authors;
+  private List<Review> reviews;
+  private double avgRating;
+
+  /**
+   * Creates a new Book object for every new book for cases when the user wants to provide
+   * only the title and author of the book.
+   */
+  public Book(String title, List<String> authors) {
+    this.id = UUID.randomUUID();
+    this.title = title;
+    this.authors = authors;
+  }
+
+  /**
+   * Creates a new Book object for every new book for cases when the user wants to provide
+   * the complete description of the book.
+   */
+  public Book(String title, List<String> authors, List<Review> reviews) {
+    this.id = UUID.randomUUID();
+    this.title = title;
+    this.authors = authors;
+    this.reviews = reviews;
+    this.avgRating = calcAvgRating(reviews);
+  }
+
```

Comment:
@jannatbedi I think we also need a constructor method that simply takes in all the Book attributes as parameters. It's for the getBook method in Datastore where you basically have to create a Book instance from a Book Entity stored in the Datastore.
```suggestion
  public Book(UUID id, String title, List<String> authors, List<Review> reviews, double avgRating) {
    this.id = id;
    this.title = title;
    this.authors = authors;
    this.reviews = reviews;
    this.avgRating = avgRating;
  }
```